Makefile

```diff
@@ -17,42 +15,36 @@ dev:
 livereload:
 	$(MANAGE) livereload $(IP):$(PORT)
 
-.PHONY: venv
-venv:
-	python3 -m venv $(VENV)
-	@echo "When developing, activate the virtualenv with 'source .venv/bin/activate' so Python can access the installed dependencies."
+.PHONY: conda-scratch
```

Comment:
Not sure why this is needed, the default option is all yeses anyways? The other command works fine for me, but maybe it breaks on a fresh install of conda?

Reply:
As noted in other command, plan "-y" says yes to everything

So it will do Yes Install, and also Yes (fresh scratch) Reinstall for someone who isn't careful

Reply:
See https://github.com/compserv/hknweb/pull/491#discussion_r990499490